clustered/ehcache-clustered/src/assemble/docker/server/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM azul/zulu-openjdk-alpine:17-jre-latest
+
+RUN apk update && apk upgrade && apk add bash
+RUN mkdir /terracotta && chgrp -R 0 /terracotta && chmod -R g+rX /terracotta
```

Comment:
Would it be simpler to mirror the structure of the existing internal Docker images here?